draft-ietf-scitt-architecture.md

```diff
@@ -547,18 +547,17 @@ It indicates the Signed Statement is securing a JSON content type, and identifyi
 
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more Issuers.
-Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST syntactically validate the Issuer's identity Claims, which may be different than the Client identity.
+1. **Client authentication:** A Client authenticates with the Transparency Service before registering Signed Statements on behalf of one or more Issuers.
+Authentication and authorization are implementation-specific and out of scope of the SCITT architecture.
+1. **Issuer Verification:** The Transparency Service MUST perform signature verification, as defined in {{Section 4.4 of RFC9052}}, and MAY perform additional checks as part of its Registration Policy.
 1. **Signature verification:** The Transparency Service MUST verify the signature of the Signed Statement, as described in {{RFC9360}}, using the signature algorithm and verification key of the Issuer.
-1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers listed above.
-The Transparency Service MAY verify the Statement payload format, content and other optional properties.
+1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers.
+The Transparency Service MAY validate the Signed Statement payload in order to enforce domain specific registration policies that apply to specific content types.
```

Comment:
Proposing removing line 555 as a SCITT Instance MAY do anything that's not explicitly called out as MUST NOT in the draft. 
@rjb4standards, this means your comment is still valid, without this sentence, and we can create more brevity.
```suggestion
1. **Signed Statement validation:** The Transparency Service MUST check that the Signed Statement includes the required protected headers.
```